spec/unit/enforcement_spec.rb

```diff
@@ -1,15 +1,29 @@
 # frozen_string_literal: true
+class MyController # < ApplicationController
```

Comment:
Are you actually worried about namespace collision with another `MyController`?  From where?  I don't get the concern as a practical matter.  If I move the class into `spec/support/` or `spec/helpers/` what has changed?  And yet defining test classes there is commonplace enough.  CurationConcerns defines classes in spec **19** times.  I think we can handle this tiny one here.